designcompose/src/main/java/com/android/designcompose/DesignSwitcher.kt

```diff
@@ -51,6 +51,7 @@ private interface DesignSwitcher {
             "#NodeNamesCheckbox",
             "#MiniMessagesCheckbox",
             "#ShowRecompositionCheckbox",
+            "#UseLocalStringResCheckbox",
```

Comment:
I don't have this checkbox when I run. I see you made a branch of the Design Switcher Figma file. I think you need to merge it to main, then fetch the `.dcf` file again.

Reply:
Done.